algorithms/linfa-svm/src/solver_smo.rs

```diff
@@ -905,39 +907,43 @@ impl<'a, F: Float, K: 'a + Permutable<F>> SolverState<'a, F, K> {
     }
 }
 
-/*
 #[cfg(test)]
 mod tests {
+    use super::{SolverParams, SolverState};
     use crate::permutable_kernel::PermutableKernel;
-    use super::{SolverState, SolverParams, SvmBase};
-    use ndarray::array;
-    use linfa_kernel::{Kernel, KernelInner};
+    use crate::SeparatingHyperplane;
 
-    /// Optimize the booth function
     #[test]
-    fn test_booth_function() {
```

Comment:
???

Reply:
I'm not sure what is the comment here :D Whole test was commented, I just added the autotraits test, and move the commented test down the lines. Let me know what should be fixed here

Reply:
Oh I didn't see that the test was already commented.